### ISSUE TITLE
postfixadmin version differs

### ISSUE THREAD

**Issue by erdoking** [BigRedS/postfixadmin-installer#49]
You wrote in your readme
"Postfixadmin **2.3.6** is downloaded from Sourceforge"

but if i look into the quellcode: (Debian Stretch)
```
my $postfixadminURL = "https://netcologne.dl.sourceforge.net/project/postfixadmin/postfixadmin/postfixadmin-3.0.2/postfixadmin-3.0.2.tar.gz";
my $postfixadminURL = 'https://downloads.sourceforge.net/project/postfixadmin/postfixadmin/postfixadmin-3.1/postfixadmin-3.1.tar.gz';
```

Newest version: **3.2**

This looks confusing ;)

maybe you just want to download the newest version?
`https://sourceforge.net/projects/postfixadmin/files/latest/download`

**Comment by jpptinsley** [comment]
Trouble with downloading the 'latest' version is that sometimes they make changes that break the installer, as they did with the setup password complexity requirements

**Comment by BigRedS** [comment]
Yeah, the other issue is that to track 'latest' requires parsing the sourceforge page to figure out what that is. There might be a better way now theyr'e on Github, too. 

I think tracking an idea of 'latest' would be a good idea, but that's a different problem to not actually installing the version claimed! 

I'm going to fix this for now by not making that claim in the readme doc, and _also_ fix up the code to be less confusing. Tracking latest can be a problem for another day.
